queries.sql

```diff
@@ -5,4 +5,27 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE neutered_weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE neutered_weight_kg >= 10.4 AND neutered_weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE neutered_weight_kg >= 10.4 AND neutered_weight_kg <= 17.3;
+UPDATE animals SET species = 'digimon' where name like '%mon';
+UPDATE animals SET species = 'pokemon' where species IS NULL OR species = '';
+SELECT name, species FROM animals;
+COMMIT;
+SELECT name, species FROM animals;
+BEGIN; 
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
```

Comment:
- [optional] Please kindly check if the data was deleted as expected after the delete command. 👍 
```suggestion
BEGIN; 
DELETE FROM animals;
SELECT * FROM animals;
ROLLBACK;
SELECT * FROM animals;
```